Sources/WordPressData/Objective-C/PostHelper.m

```diff
@@ -255,4 +260,168 @@ + (NSArray *)mergePosts:(NSArray <RemotePost *> *)remotePosts
     return posts;
 }
 
++ (ReaderPost *)createOrReplaceFromRemotePost:(RemoteReaderPost *)remotePost
```

Comment:
Good idea to just move it to `PostHelper` to unblock the rest of the work. It seems like it would be a bit challenging to update this.

Reply:
I'm hoping that we can keep this class as Objective-C, like having two modules WordPressData and WordPressDataObjC, similar to a few other libraries. We'll see how it goes.